src/getdata.js

```diff
@@ -0,0 +1,11 @@
+const key = 'bdc41b55a4c712c6598fce628e1a5254';
+const url = 'https://api.openweathermap.org/data/2.5/weather?q=';
```

Comment:
- [ ] [OPTIONAL] When the project is complete consider storing the key in an env file for security reasons. The url could be in a constants file or in the same env too.
Interesting reading on how to achieve this using webpack: https://medium.com/@justintulk/passing-environment-variables-into-your-code-with-webpack-cab09d8974b0

Reply:
### I will do that, thank you for the note.